Infrastructure-Scripts/Import-ApimSwaggerApiDefinition.ps1

```diff
@@ -76,6 +76,7 @@ function Add-AppServiceWhitelist ($AppServiceResourceGroup, $AppServiceName) {
         Write-Output "Whitelisting $MyIp"
         $Priority = ($IpRestrictions.MainSiteAccessRestrictions | Where-Object { $_.Action -eq "Allow" }).Priority[-1] + 1
         $null = Add-AzWebAppAccessRestrictionRule -ResourceGroupName $AppServiceResourceGroup -WebAppName $AppServiceName -Name "DeployServer" -IpAddress "$MyIp/32" -Priority $Priority -Action Allow
+        Start-Sleep -Seconds 5
```

Comment:
Could this be -AsJob and then a Wait-Job instead of sleep?

Reply:
The Add-AzWebAppAccessRestrictionRule cmdlet completes, the sleep is needed so the whitelisted IP is in effect.

Added to PR description